### ISSUE TITLE
not seeing job descriptions

### ISSUE THREAD

**Issue by MaxPleaner** [grnhse/greenhouse_io#25]
I am using `client.jobs(status: 'true')` but the `status: 'true'` part doesn't seem to change anything.

```ruby
job.keys
=> [:id,
 :name,
 :requisition_id,
 :notes,
 :status,
 :created_at,
 :opened_at,
 :closed_at,
 :departments,
 :offices,
 :hiring_team,
 :custom_fields,
 :openings]
```

This is using Harvester by the way

**Comment by capablemonkey** [comment]
Hi @MaxPleaner,

The GET `/v1/jobs` doesn't support filtering jobs by their `status`.  See the available querystring options here: https://developers.greenhouse.io/harvest.html#list-jobs

In any case, to pass options in, you'd want to do so like this:

```
client.jobs(nil, per_page: 3)
```

The first parameter is the Job ID which we set to `nil` since we want to list all jobs.  In this example, we set `per_page` to 3 to get 3 jobs per call.

Hope that helps!
